server/cpho/jinja2/indicators/upload_indicator.jinja2

```diff
@@ -208,32 +211,49 @@
                       </ul>
                     </div>
                     <div class="p-2 border">
-                      {{ tdt("Region:") }}
+                      <span class="fw-bold">{{ tdt("Region:") }}</span>
+                      <hr>
                       <ul>
                         <li>{{ tdt("ATLANTIC") }}</li>
                         <li>{{ tdt("PRAIRIE") }}</li>
                         <li>{{ tdt("TERRITORIES") }}</li>
                       </ul>
                     </div>
                     <div class="p-2 border">
-                      {{ tdt("Sex:") }}
+                      <span class="fw-bold">{{ tdt("Sex:") }}</span>
+                      <hr>
                       <ul>
                         <li>{{ tdt("MALES") }}</li>
                         <li>{{ tdt("FEMALES") }}</li>
                       </ul>
                     </div>
                     <div class="p-2 border">
-                      {{ tdt("Canada:") }}
+                      <span class="fw-bold">{{ tdt("Canada:") }}</span>
+                      <hr>
                       <ul>
                         <li>{{ tdt("CANADA") }}</li>
                       </ul>
                     </div>
                     <div class="p-2 border">
-                      {{ tdt("Age Group:") }}
+                      <span class="fw-bold">{{ tdt("Age Group:") }}</span>
+                      <hr>
                       <ul>
                         <li>{{ tdt("Open Text Field") }}</li>
                       </ul>
                     </div>
+                    <div class="p-2 border">
+                      <span class="fw-bold">{{ tdt("Living Arrangement:") }}</span>
+                      <hr>
+                      <ul>
+                        <li>{{ tdt("MALE LIVING ALONE") }}</li>
+                        <li>{{ tdt("FEMALE LIVING ALONE") }}</li>
+                        <li>{{ tdt("COUPLE NO CHILDREN") }}</li>
+                        <li>{{ tdt("COUPLE WITH CHILD(REN) LESS THAN 18 YEARS OLD") }}</li>
+                        <li>{{ tdt("FEMALE LONE PARENT WITH CHILD(REN) LESS THAN 18 YEARS OLD") }}</li>
+                        <li>{{ tdt("MALE LONE PARENT WITH CHILD(REN) LESS THAN 18 YEARS OLD") }}</li>
+                        <li>{{ tdt("OTHER LIVING ARRANGEMENTS") }}</li>
```

Comment:
Not important for now, but these strings should probably come from the `DimensionValue` model, or at least the mapper util